lib/online_order.rb

```diff
@@ -0,0 +1,74 @@
+require_relative './order'
+require 'csv'
+require 'awesome_print'
+
+
+module Grocery
+  class OnlineOrder < Order
+
+    attr_reader :order_id, :products, :customer_id, :order_status
```

Comment:
Since `OnlineOrder` inherits from `Order` you don't need to have `order_id`, or `products`.